<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesture Recognizer</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188)</string>
   </property>
   <property name="text">
    <string>Add Gesture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emptyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Recognize</string>
   </property>
  </widget>
  <widget class="Line" name="lineTop">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="lineLeft">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>20</width>
     <height>521</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="lineBottom">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="lineRight">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>20</width>
     <height>521</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="gestureList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="gestureListLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestures</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gestureInfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Last recognized gesture:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gestureRecognizeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>- no gesture -</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188)</string>
   </property>
   <property name="text">
    <string>Delete Gesture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188)</string>
   </property>
   <property name="text">
    <string>Train Gesture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
